cloudbank-v2/spring-apps/account/pom.xml

```diff
@@ -4,90 +4,101 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
 	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
 	<modelVersion>4.0.0</modelVersion>
+
 	<parent>
-		<groupId>org.springframework.boot</groupId>
-		<artifactId>spring-boot-starter-parent</artifactId>
-		<version>2.7.12</version>
-		<relativePath/> <!-- lookup parent from repository -->
+		<groupId>com.example</groupId>
+		<artifactId>sample-spring-apps</artifactId>
+		<version>0.0.1-SNAPSHOT</version>
 	</parent>
-	<groupId>com.example</groupId>
-	<artifactId>accounts</artifactId>
+
+	<artifactId>account</artifactId>
 	<version>0.0.1-SNAPSHOT</version>
-	<name>accounts</name>
-	<description>Demo project for Spring Boot</description>
+	<name>account</name>
+	<description>Account Application</description>
+
 	<properties>
-		<java.version>17</java.version>
+		<microprofile-lra-api.version>1.0</microprofile-lra-api.version>
+		<narayana-lra.version>5.13.1.Final</narayana-lra.version>
+		<jakarta.enterprise.cdi-api.version>2.0.2</jakarta.enterprise.cdi-api.version>
+		<oracle-springboot-starter.version>2.7.7</oracle-springboot-starter.version>
+		<javax.el.version>3.0.0</javax.el.version>
 		<liquibase.version>4.19.0</liquibase.version>
 	</properties>
+
 	<dependencies>
-	    <dependency>
-            <groupId>org.springframework.boot</groupId>
-            <artifactId>spring-boot-starter-actuator</artifactId>
-        </dependency>
-        <dependency>
-            <groupId>io.micrometer</groupId>
-            <artifactId>micrometer-registry-prometheus</artifactId>
-        </dependency>
 		<dependency>
 			<groupId>org.springframework.boot</groupId>
-			<artifactId>spring-boot-starter-web</artifactId>
+			<artifactId>spring-boot-starter-data-jpa</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>org.springframework.cloud</groupId>
+			<artifactId>spring-cloud-starter-netflix-eureka-client</artifactId>
+			<exclusions>
+				<exclusion>
+					<groupId>javax.ws.rs</groupId>
+					<artifactId>jsr311-api</artifactId>
+				</exclusion>
+				<exclusion>
+                    <groupId>javax.servlet</groupId>
```

Comment:
can we fix the formatting?

Reply:
Formatting looks good in VS COde and IntelliJ, not sure how to fix @markxnelson